agent/util-scripts/tool-meister/tool-data-sink-ep

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+_dir="$(dirname ${0})"
+
+source /etc/profile.d/pbench-agent.sh
+source /opt/pbench-agent/base
+# Instruct the Tool Data Sink not to daemonize.
+/opt/pbench-agent/util-scripts/tool-meister/pbench-tool-data-sink "${REDIS_HOST}" "${REDIS_PORT}" "${PARAM_KEY}" no
```

Comment:
Unfortunately, `_dir` and the `source` invocations mimic the environment that all the pbench-agent `util-scripts` expect.  So that needs to be provided by these files.  And I'd rather keep these concerns here in a `bash` file than in a `Dockerfile` to avoid restrictions.

Reply:
But...does `pbench-tool-data-sink` actually use `_dir` or anything that the `source` invocations provide?  If not, then we _could_ get by without them here...in which case, we __could__ invoke `pbench-tool-data-sink` directly using the `Dockerfile` instead of adding the clutter and complication of a wrapper script that we don't need.